app/dot/cpp/repository/repository/BaseRepository.java

```diff
@@ -123,8 +125,8 @@ public List<T> listHistoryRecords(String id) {
     final var historyEntities = new ArrayList<T>();
 
     getHistoryCollection()
-        .find(eq(RECORD_ID, id))
-        .sort(descending(TIMESTAMP))
+        .find(Filters.eq(RECORD_ID, id))
```

Comment:
let's just use eq all around this class of full name with class. i see in some places you removed the class but here it was added

Reply:
These are Mongo driver Filters and Sorts, not Morphia, that's why they are written differently. I decided to import the Morphia ones statically and these as normal imports.